### ISSUE TITLE
Pending Wars screen

### ISSUE THREAD

**Issue by jiakeliu** [gunnertech/thumbwar-ios-client#96]
This screen houses all wars that you initiated but haven't been answered by the opponent. Only you can access this screen via the Profile page. 

The layout of this screen should look like the Alerts screen.

For each "alert" or pending war, display the following: 

Body text: "You challenged Cody (opponent name is bolded) to a bet"
Button text: "Send Reminder" (this button can be tapped once every 24 hours. After it's tapped, the button becomes inactive and the text is changed to "Reminder Sent". Cody receives a push notification: "Adam sent you a reminder for your bet" and simply surfaces the alert to the top of the alert page)
"X" button: when tapped, ask the user on a modal popup:
"Are you sure you want to cancel this challenge?" 
Green Button "Yes" 
Red Button "No"

Mockup to follow.

**Comment by rand027** [comment]
Would something like this work? I think we can lose the "to a bet" and just have it read "you challenged cody." It doesn't add much to the description and it will help us keep everything on one line. I also shortened the button text, thoughts?
![yourbets](https://cloud.githubusercontent.com/assets/11729006/7481488/14d2e886-f325-11e4-86a5-9046ca72b149.jpg)

**Comment by jiakeliu** [comment]
I like how simple this looks, but how would a user cancel the bet?

**Comment by rand027** [comment]
Good question. I saw the "x" request in your feedback above, but I assumed it was for the notification. We have a few options. The "sent" state could me a temporary animation replaces by a cancel button. We could make the alerts full width and make the reminder/close options buttons that you have to swipt to reveal (i.e. mail on an iphone) or we could use an icon for sending a reminder and pop a modal with send/cancel confirmation.

**Comment by adamespn** [comment]
Why not add these to the existing alerts screen?

**Comment by jiakeliu** [comment]
@adamespn I think pending wars that require action from your opponent shouldn't take up the prime space in Alerts, which is reserved for items that need your immediate attention or are very interesting to you. Since our Profile screen is already displaying all types of wars (Win, Lose, Active), I think it's a good place to house the "Pending" wars, but it should be tucked away and can only be accessed via the ellipses.

@rand027 I like the first option of the swipe action. I just wonder if that's intuitive enough? I don't think we should go with the modal pop up though.

**Comment by jiakeliu** [comment]
@rand027 can we have a mock up of the slide to cancel screen? Can you also apply the same design to the alert screen here: #20

**Comment by adamespn** [comment]
Can we put these in the existing alerts screen instead of creating whole new screen?

**Comment by adamespn** [comment]
@CodySwannGT can we support a new server side push use case in this iteration?

**Comment by CodySwannGT** [comment]
@adamespn no. Looking at this screen requires a bunch of new server functionality.

I'm voting that this is not MVP

**Comment by CodySwannGT** [comment]
In the meantime, I would suggest the functionality be accomplished this way:

1) I challenge Adam to a ThumbWar
2) Adam doesn't respond
3) I go into my ThumbWars and see Adam hasn't responded
4) I comment on the War "C'mon wuss!"

It accomplishes the same functionality, but doesn't require at minimum two iterations of work to do.

**Comment by jiakeliu** [comment]
@CodySwannGT do you receive a push notification when a user comments on your thumbwar?

**Comment by CodySwannGT** [comment]
yeah.

**Comment by husseinm** [comment]
Do we want to do this? Currently all current wars are visible via a users profile... Status?

**Comment by jiakeliu** [comment]
This was intended for wars that have not been accepted nor rejected. But since the backend doesn't support it yet, let's put this in the Future Considerations milestone for now.
